roles/edpm_bootstrap/defaults/main.yml

```diff
@@ -23,6 +23,7 @@ edpm_bootstrap_packages_bootstrap:
   - lvm2
   - crudini
   - jq
+  - NetworkManager-ovs
```

Comment:
Does the linux-system-role not make sure this is installed? If not, would that be considered a bug?
I'd move the package install into edpm_network_config when the system role is used.
And move the os-net-config package install into edpm_network_config and only installed when os-net-config is used.

Reply:
We may want to keep a list of those. Maybe a jira task under some general NG topic?

Reply:
Do you mean a list of required packages?

Reply:
Yeah. I mean we kind of do so already, but this one would be attached to a task to make their installation automated.